src/components/BSelectOption/BSelectOption.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <option v-bind:value=option.id>
+        {{ option.text }}
```

Comment:
I think we should make these slots instead of passing props around all over the place.